src/main/java/com/commercetools/sync/inventories/InventorySyncOptions.java

```diff
@@ -38,8 +38,8 @@
     }
 
     /**
-     * @return option that indicates whether sync process should create supply channel of given key when it doesn't
-     *      exists in a target system yet.
+     * @return option that indicates whether sync process should create a supply channel of given key when it doesn't
```

Comment:
whether sync process should create a supply channel of given key -> whether **the** sync process should create a supply channel of **the** given key

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/319/commits/ac7e561a12b70d8d3c43ae1f8daf95f608542d38